src/main/java/org/commonjava/indy/service/archive/config/ConfigResource.java

```diff
@@ -1,4 +1,4 @@
-package org.commonjava.indy.service.config;
+package org.commonjava.indy.service.archive.config;
 
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 
```

Comment:
I think this is just setup resource from quarkus, so it should not be needed anymore.